draft-ietf-quic-transport.md

```diff
@@ -2692,10 +2708,19 @@ implementation decision, and an implementation should be careful to delay
 conservatively, since any delay is likely to increase application-visible
 latency.
 
-Regular QUIC packets are "containers" of frames; a packet is never retransmitted
-whole.  How an endpoint handles the loss of the frame depends on the type of the
-frame.  Some frames are simply retransmitted, some have their contents moved to
-new frames, and others are never retransmitted.
+Regular QUIC packets are "containers" of frames.  When an endpoint receives an
+ACK frame for one or more transmitted packets, all frames in the acknowledged
+packets are considered to have been received and processed by the peer, with one
+exception.  A PATH_CHALLENGE frame (see {{frame-path-challenge}}) is used to
+validate a peer's ownership of its address.  An ACK frame received for a
+PATH_CHALLENGE frame is not adequate to indicate that the PATH_CHALLENGE was in
+fact received.  A PATH_CHALLENGE is considered acknowledged only when the
+corresponding PATH_RESPONSE (see {{frame-path-response}}) is received for it.
```

Comment:
I still think this new text is more confusing than helpful.  You're going to have to write some new text that's more complete than this for connection migration, so I would keep this section as is for now.

Reply:
as is = as it was

Reply:
Removed offending text. Many things need fixing in migration.